testnets/empetestnet/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "empetestnet",
+  "assets": [
+    {
+      "description": "Empe End-to-End Verifiable Data Infrastructure",
+      "extended_description": "Empeiria is the first End-to-End Verifiable Data Infrastructure (EVDI). It enables seamless web3 adoption through one-click deployment, empowering organizations with the data of the future",
+      "denom_units": [
+        {
+          "denom": "uempe",
+          "exponent": 0
+        },
+        {
+          "denom": "empe",
+          "exponent": 6
+        }
+      ],
+      "base": "uempe",
+      "display": "uempe",
```

Comment:
display should likely be "empe", not "uempe"

Reply:
ok done